### ISSUE TITLE
how can I get category feed?

### ISSUE THREAD

**Issue by dracodoc** [wmeints/hexo-generator-feed-per-category#4]
@wmeints I followed the instructions and I can see the RSS feed, but how can I get a feed for specific category? There is only one feed file, and I saw all the posts in feed.

**Comment by dracodoc** [comment]
@wmeints [here is my blog site](http://dracodoc.github.io/). You can check the side bar menu and see the RSS feed. There are categories in the RSS feed, but I don't know how to get a specific feed with only one category, such as category `R`.

**Comment by wmeints** [comment]
This generates a feed for every category you have. It doesn't support generating a feed for just one category. If you have a category `R` the feed is generated as `categories/R/feed.xml`. It will contain all the posts that have the category R attached to them. Keep in mind though that if one post has multiple categories it will appear in multiple feeds.

**Comment by dracodoc** [comment]
Thanks for the tip! I found a `rss2.xml` under `categories/R` folder, even if I set the feed type as `atom`. The next theme I used seemed only work with atom feed. With feed type set as `atom`, I still got category feed in `rss2`, but it didn't include the full text like the `atom.xml`. 

I can understand the reason to not include full text for category feed, though the R-bloggers.com I wanted to submit with my blog ask for full text feed with category.
